doc/build.md

```diff
@@ -50,11 +50,13 @@ Then, we can build and run tests.
 
 ```bash
 go generate ./...
-SQLFLOW_TEST_DB=mysql go test -v -p 1 ./...
+PYTHONPATH=/sqlflow/python SQLFLOW_TEST_DB=mysql go test -v -p 1 ./...
```

Comment:
Should be `PYTHONPATH=$GOPATH/src/sqlflow.org/sqlflow/python`, see `test_units.sh`

Reply:
Good idea! I'll issue another pull request to improve.